src/snovault/helpers/helper.py

```diff
@@ -340,12 +340,16 @@ def build_aggregation(facet_name, facet_options, min_doc_count=0):
     agg_name = facet_name.replace('.', '-')
 
     facet_type = facet_options.get('type', 'terms')
-    if facet_type == 'terms':
+    if 'length' in facet_options:
+        facet_length = 3000 if facet_options['length'] == 'long' else 200
+    else:
+        facet_length = 200
+    if facet_type in ['terms', 'typeahead']:
```

Comment:
The one suggestion just if you feel like is to move the assignment on line 342 to just above 347, just so the assignment is just above where you check it. Technically, it really doesn’t matter though.